os/arch/arm/src/s5j/s5j_i2s.c

```diff
@@ -2017,8 +2023,16 @@ static int i2s_irq_handler(int irq, FAR void *context, FAR void *arg)
  *
  ****************************************************************************/
 
-struct i2s_dev_s *s5j_i2s_initialize(void)
+struct i2s_dev_s *s5j_i2s_initialize(int port)
 {
+	if (port >= CONFIG_S5J_I2S_MAXPORTS) {
+		lldbg("ERROR: Port number outside the allowed port number range\n");
+		return NULL;
+	}
+	if (g_i2sdevice[port] != NULL) {
+		return &g_i2sdevice[port]->dev;
```

Comment:
Don't you need a reference number like refs?
When several devices use i2s and one device try to uninitialize, it can do abnormal operatoin.

Reply:
There is no function which would uninitialize the device. @btheosam can clarify on this more. I personally feel there is no need to maintain refs here.